install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -69,10 +70,6 @@ spec:
               value: |
                 2.8.0=registry.redhat.io/amq7/amq-streams-kafka-28-rhel8:2.0.0
                 3.0.0=registry.redhat.io/amq7/amq-streams-kafka-30-rhel8:2.0.0
-            - name: STRIMZI_KAFKA_CONNECT_S2I_IMAGES
```

Comment:
In this file, you will need to add the metering labels as well.

Reply:
I think you mean I am missing the drain-cleaner metering labels in `install/drain-cleaner/openshift/060-Deployment.yaml`, I just added them now